test/com/facebook/buck/jvm/kotlin/testdata/kotlin_library_description/com/example/mixed/Example.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.good
```

Comment:
I can't figure out how to get tests to run locally nor even compile buck using buck. 
`./bin/buck build buck` or `./bin/buck test //test/com/facebook/buck/jvm/kotlin:kotlin` fail with
`Cannot find annotation method 'fieldVisibility()' in type 'com.fasterxml.jackson.annotation.JsonAutoDetect': class file for com.fasterxml.jackson.annotation.JsonAutoDetect not found` and some other errors as well.